nomad/state/state_store_acl_sso.go

```diff
@@ -186,10 +185,10 @@ func (s *StateStore) GetACLAuthMethodByName(ws memdb.WatchSet, authMethod string
 	return nil, nil
 }
 
-// GetDefaultACLAuthMethodByType returns a default ACL Auth Methods for a given
+// GetDefaultACLAuthMethod returns a default ACL Auth Methods for a given
 // auth type. Since we only want 1 default auth method per type, this function
 // is used during upserts to facilitate that check.
```

Comment:
I think we should update the function comment as I don't believe this is now correct based on the change of the logic.

Reply:
resolved in [a56579e](https://github.com/hashicorp/nomad/pull/15869/commits/a56579ec97e83cd84f20ab0cdbbf93374a0af948)